plugins/llvm/llvm_binary.hpp

```diff
@@ -392,17 +295,23 @@ std::vector<symbol> readPE32(const COFFObjectFile& obj) {
             }
         }
 
-        auto addr = sec->VirtualAddress + pe32->ImageBase + sym->Value;
+        auto addr = sec->VirtualAddress + image_base + sym->Value;
         symbols.push_back(symbol(*it,addr,size));
     }
     return symbols;
 }
 
 std::vector<symbol> read(const COFFObjectFile& obj) {
     if (obj.getBytesInAddress() == 4) {
-	return readPE32(obj);
+	const pe32_header *pe32;
```

Comment:
this is a copy-paste of https://github.com/BinaryAnalysisPlatform/bap/pull/517/files#diff-fe06f65ca8c62da7846dcdba4b819235R196, it definitely needs some abstraction.